source/libraries/LegacyRepMigrator.ts

```diff
@@ -98,11 +102,12 @@ export class LegacyRepMigrator {
     }
 
     private async migrateBalances(): Promise<void> {
-        const promises: Array<Promise<any>> = [];
-        for (let balancesChunk of this.chunkedBalances) {
-            promises.push(this.migrateOwners(balancesChunk));
-        }
-        await resolveAll(promises);
+        let i = 0;
+        await Parallel.each(this.chunkedBalances, async balancesChunk => {
+            await this.migrateOwners(balancesChunk);
+            i+=this.chunkSize;
+            if (i % 100 == 0) console.log(`Migrated ${i} balances`);
```

Comment:
How many batches do you suppose there will be? Randomly printing 1% will work well enough?

Reply:
Ahh, looks like plenty!